src/templates/scalebar_form.template.html

```diff
@@ -107,24 +107,43 @@
             <% } %>
         </div>
     </div>
-    <div class="row gy-2 gx-3 align-items-center" style="--bs-gutter-x: 0.3rem;">
+    <div class="row gy-2 gx-3 align-items-center" style="--bs-gutter-x: 0.3rem; margin-top: 0px;">
         <div class="col-auto input-group-sm">
-            <label class="col-form-label col-form-label-sm" style="width:50px">Height</label>
+            <label class="col-form-label col-form-label-sm" style="width:40px">Height</label>
         </div>
         <div class="form-group col-auto scalebar_length input-group-sm">
-            <input type="number" class="label-text scalebar-height form-control input-sm" 
-            placeholder="Height" value="<%= height %>" style="width:52px" />
+            <input type="number" class="label-text scalebar-height form-control input-sm"
+            placeholder="Height" value="<%= height %>" style="width:30px" />
         </div>
-        <div class="col-auto input-group-sm" style="margin-right: 10px;">
+        <div class="col-auto input-group-sm" style="margin-right: 0px; padding-left: 0px;">
             <label class="col-form-label col-form-label-sm">px</label>
         </div>
 
-        <div class="col-auto input-group-sm">
+        <div class="col-auto input-group-sm" style="margin-left: 5px;">
+            <label class="col-form-label col-form-label-sm" style="width:40px">Margin</label>
+        </div>
+        <div class="form-group col-auto scalebar_length input-group-sm">
+            <input type="number" class="label-text scalebar-margin form-control input-sm"
+            placeholder="Margin" value="<%= margin %>" style="width:38px; padding-left: 3px; padding-right: 3px;" />
```

Comment:
I found that we can reduce the width of this input to `30px` and it's still wide enough for 2 digits. This fixes an issue I'm seeing with the row being too long and overflowing:

<img width="621" height="550" alt="Screenshot 2025-07-28 at 16 34 18" src="https://github.com/user-attachments/assets/dcc1a877-bea9-4c94-b31a-1ee3f93d0437" />